app/src/main/java/com/onboarding/parkingsystemkotlin/mvp/view/base/ActivityView.kt

```diff
@@ -0,0 +1,12 @@
+package com.onboarding.parkingsystemkotlin.mvp.view.base
+
+import android.app.Activity
+import java.lang.ref.WeakReference
+
+open class ActivityView(activity: Activity) {
+    private val activityRef: WeakReference<Activity> = WeakReference<Activity>(activity)
+
+    fun getActivity(): Activity? {
+        return activityRef.get()
+    }
```

Comment:
here instead of return you can use 
fun getActivity(): Activity? = activityRef.get()
to reduce lines

Reply:
done